tests/clv/models/test_beta_geo.py

```diff
@@ -189,7 +280,7 @@ def test_expected_probability_alive(self):
                 "r": rng.normal(r, 1e-3, size=(2, 25)),
             }
         )
-        bg_model._fit_result = fake_fit
+        bg_model.fit_result = fake_fit
```

Comment:
I think the equivalent to the old test approach is to override `self.idata`, directly which won't trigger any of the checks or warnings in the `setter` method.

This would apply to other fake fit tests

Reply:
replaced with model.idata = fake_fit